client/providers/mocks/provider.go

```diff
@@ -0,0 +1,146 @@
+// This code was originally generated by mockery v2.32.0 and modified
+
+package mocks
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/json"
+	"time"
+
+	"github.com/awnumar/memguard"
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jws"
+	"github.com/openpubkey/openpubkey/verifier"
+	"github.com/stretchr/testify/mock"
+)
+
+// OpenIdProvider is an autogenerated mock type for the OpenIdProvider type
```

Comment:
What tool/command did you use to generate this? A comment describing how to generate this would be useful if anyone needs to regenerate this code